src/pybind/mgr/dashboard/README.rst

```diff
@@ -55,6 +55,16 @@ See the `Ceph Dashboard plugin documentation
 and configure the dashboard manually and how to configure other settings, e.g.
 access to the Ceph object gateway.
 
+Supported browsers
+------------------
+
+We support `Chrome 68+ <https://www.google.com/chrome/>`_ and
```

Comment:
I'm a bit wary about the term "support" here - what does this mean?

I wonder if it'd be more appropriate to simply state the browser versions that the dashboard is known to work with. Would it be prudent to mention the browsers' OS platforms as well?
I'd hope that this should not make a difference.

"The Ceph Manager Dashboard" is tested and developed on the following browsers..."